bashrc.d/zz-10-base.sh

```diff
@@ -344,10 +344,16 @@ fi
 if [[ ${TERM:-} == dumb ]]; then
     export EDITOR=$(type -p emacsclient)
     export NODE_NO_READLINE=1
-    export PAGER=cat
+    export PAGER=$(type -p cat)
     export SYSTEMD_COLOR=0
 fi
-
+# See above for these
+if [[ ! ${EDITOR:-} ]]; then
+    unset EDITOR
```

Comment:
Do you ever install home-env alongside some other configuration (ex at NERSC)? I'm thinking there could be a situation with no less/emacs where you'd want to revert to the system defaults instead of unset.

Reply:
That's a good point. f54a57d